docs/buildtest_perlmutter.rst

```diff
@@ -0,0 +1,116 @@
+Buildtest Tutorial on Perlmutter
+===================================
+
+This tutorial will be conducted on `Perlmutter <https://docs.nersc.gov/systems/perlmutter/>`_ system. If you need account access please
+`obtain a user account <https://docs.nersc.gov/accounts/>`_.
+
+Setup
+------
+
+Once you have a NERSC account, you can `connect to NERSC system <https://docs.nersc.gov/connect/>`_. You will need access to a
+terminal client and ssh into perlmutter as follows::
+
+    ssh <user>@perlmutter-p1.nersc.gov
+
+To get started please load the **python** module since you will need python 3.7 or higher to use buildtest. This can be done by running::
+
+    module load python
+
+Next, you should :ref:`Install buildtest <installing_buildtest>` by cloning the repository in your $HOME directory.
+
+Once you have buildtest setup, please clone the following repository https://github.com/buildtesters/buildtest-nersc in your $HOME directory as follows::
```

Comment:
Missing instructions to setup buildtest

Reply:
Suggestion: 'please clone the following repository into your home directory:'

Since the git command has the full repo you don't need to list it twice.

Reply:
yeah so the setup requires them to read the [Installing buildtest](https://buildtest.readthedocs.io/en/devel/installing_buildtest.html) page, this includes creating a python virtual environment and sourcing the setup script. Instead of redocumenting i just put link to page.